packages/design-system/src/components/FilterChip/FilterChip.test.tsx

```diff
@@ -0,0 +1,91 @@
+import FilterChip from './FilterChip';
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+describe('FilterChip', () => {
+  const renderFilterChip = (customProps = {}) => {
+    const defaultProps = {
+      label: 'Foo',
+      onDelete: () => {},
+    };
+    const props = { ...defaultProps, ...customProps };
+    return render(<FilterChip {...props} />);
+  };
+
+  it('should include children as label', () => {
+    renderFilterChip();
+    const chipEl = screen.getByRole('button');
+    expect(chipEl.textContent).toEqual('Foo. Remove Foo filter .');
+
+    expect(chipEl.innerHTML).toMatchInlineSnapshot(
```

Comment:
What's the benefit of this being inline vs in a snapshot file?

Reply:
Not sure if there's a benefit. Is it harder to read if it's inline?

Reply:
All our other snapshots are in separate files, so I was wondering if this was a special case for a hidden reason. I do think it's a little easier, but I'm mainly interested in consistency